.github/maintainers_guide.md

```diff
@@ -97,7 +105,7 @@ Releasing can feel intimidating at first, but rest assured: if you make a mistak
 
     - **External**
 
-      - **Bot Developer Hangout** (`dev4slack.slack.com`) in **#slack-api**
+      - **Slack Community Hangout** (`community.slack.com/`) in **#lang-javascript**. Include a link to the package on NPM as well as the release notes.
```

Comment:
Can we link to a previous announcement in #lang-javascript here, i.e. my last announcement there: https://community.slack.com/archives/CHF1FKX4J/p1657293144932579